### PR TITLE
Added read order endpoint

### PR DESCRIPTION
**Pull request by Momocky**
Made the following changes:
-Added read order endpoint in "routes.py"
-Added test case for read endpoint

### CODE REVIEW

**Comment by j5shan** [comment]
test failed -AttributeError: 'OrderTestSuite' object has no attribute '_create_orders'.  Added in issue #24, please review and merge previous issue to resolve.

**Comment by j5shan** [comment]
Also please uncomment parts in code that relies on get/read method, and grammar check for "an order" instead of "a order"

**Comment by j5shan** [comment]
latest commit #da0b175 has syntax error cannot  run

**Comment by j5shan** [comment]
Please merge master into branch

File: tests/test_routes.py [hunk 1]
```diff
@@ -164,6 +163,28 @@ def test_update_order(self):
         self.assertEqual(updated_order["address"], "unknown")
         self.assertEqual(updated_order["customer_id"], 0)
 
+        
+    # ----------------------------------------------------------
+    # TEST READ
+    # ----------------------------------------------------------
+    def test_get_order(self):
+        """It should Get a single Order"""
+        get the id of an order
```
Comment (myl525) [inline tests/test_routes.py]: this line should be deleted or commented

**Pull request closed without merging**